src/shared/utilities/vsCodeUtils.ts

```diff
@@ -90,6 +117,24 @@ export function showInstallExtensionMsg(
     return false
 }
 
+export function buildMissingExtensionMessage(
+    extId: string,
+    extName: string,
+    minVersion?: string,
+    feat = `${getIdeProperties().company} Toolkit`
+): string {
+    const expectedVersionMsg = minVersion ? ` of version >=${getSemver(minVersion)}` : ''
```

Comment:
`getSemver` can throw here, which is not really helpful since we are already in the middle of handling an error.

It also isn't useful for `getSemver` to throw from `isExtensionInstalled`, instead just handle the nullish result in `isExtensionInstalled` and treat it as an invalid  version.

IOW: `getSemver` isn't needed, it can be elimiated. Just use `semver.coerce` directly.